code/modules/admin/verbs/freeze.dm

```diff
@@ -28,7 +28,7 @@ var/global/list/frozen_mob_list = list()
 
 /mob/living/proc/admin_Freeze(var/client/admin, skip_overlays = FALSE)
 	if(istype(admin))
-		to_chat(src, "<b><font color= red>You have been frozen by [key_name(admin)]</b></font>")
+		to_chat(src, "<b><font color= red>You have been frozen by [usr]</b></font>")
```

Comment:
I think the admins want their ckeys to be shown NOT their characters, replace usr with just admin and that should do the trick since admin is being passed as a client anyway

Reply:
oooo I completely misread what Shockpoint sent to me in Discord, big rip, my bad, will be adjusted